app/controllers/api/v1/customer_payment_method_controller.rb

```diff
@@ -0,0 +1,77 @@
+module Api
+  module V1
+    class CustomerPaymentMethodController < Api::V1::ApiController
```

Comment:
O que acha de padronizar com as demais classes ? 💅🏾 
```suggestion
class Api::V1::CustomerPaymentMethodController < Api::V1::ApiController
```